packages/playwright-core/src/utils/linuxUtils.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ * Modifications copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs';
+
+let osRelease: {
+  id: string,
+  version: string,
+} | undefined;
+
+export async function getLinuxDistributionInfo(): Promise<{ id: string, version: string } | undefined> {
+  if (process.platform !== 'linux')
+    return undefined;
+  if (!osRelease) {
+    try {
+      // List of /etc/os-release values for different distributions could be
+      // found here: https://gist.github.com/aslushnikov/8ceddb8288e4cf9db3039c02e0f4fb75
+      const osReleaseText = await fs.promises.readFile('/etc/os-release', 'utf8');
```

Comment:
What happened to `'/etc/upstream-release/lsb-release'`?

Reply:
This was added in 2021 to support Linux Mint: https://github.com/microsoft/playwright/pull/6569

However, I don't see this file existing in any of the mint docker images I tried; e.g.:

```bash
$ docker run --platform linux/amd64 --rm -it vcatechnology/linux-mint cat /etc/upstream-release/lsb-release
cat: /etc/upstream-release/lsb-release: No such file or directory
```

So I'd remove this and wait for someone from Linux Mint to re-appear.